src/markerclusterer.ts

```diff
@@ -180,8 +180,40 @@ export class MarkerClusterer extends OverlayViewSafe {
 
       // allow algorithms to return flag on whether the clusters/markers have changed
       if (changed || changed == undefined) {
+        const records = new Map<{ lat: number; lng: number }, Cluster>();
+
+        this.clusters.forEach(
+          (cluster) =>
+            cluster.marker &&
```

Comment:
not sure if there is any coding standards in this repo (@amuramoto ?) but personally prefer the longer `if () {...}` version

Reply:
eslinted.